README.md

```diff
@@ -10,33 +10,63 @@ TODO: Write description
 
 Let's get you all setup and ready to go with _The Agriculture Core_!
 
+## Configuration
+The agriculture-core pulls its configuration from a `.env` file at the root of the project. These configuration variables are loaded into `process.env.<VAR_NAME>` and are accessible accross the application. To get started, you will have to create this configuration file using the following format:
+
+```
+# Application Configuration
+PORT=
+
+# Database Environment
+DB_HOST=
+DB_NAME=
+DB_USER=
+DB_PASS=
+
+# Authentication
+JWT_SECRET=
+JWT_ISSUER=
+JWT_AUDIENCE=
+JWT_EXPIRES=
+```
+
 ## Dependencies
 
 Well of course you need Node! You can get this [here](https://nodejs.org/en/)!
```

Comment:
I'd like to keep our documentation in a professional tone and in third person. 

Ticket made https://github.com/agriculture-capstone/agriculture-core/issues/7

Reply:
I'd personally rather avoid overly professional tones in the documentation we leave behind. A more casual tone explains the topic just as well, while conveying a more soothing message to people using this to learn. More professional README's can be pretty intimidating. I think we should be judging a README based on how effectively it can teach someone what it intends to

Reply:
Moving the response to #7